framework/wazuh/core/indexer/base.py

```diff
@@ -1,9 +1,27 @@
 import logging
+from enum import Enum
 from typing import Any, Dict, List, Tuple
 
 from opensearchpy import AsyncOpenSearch
 
 
+class Key(str, Enum):
```

Comment:
We should decide whether to use constants or an enum, I personally prefer the latter

Reply:
It's ok for me. I'd rename the class to something more specific, `QueryKey` or something like that.

Reply:
The thing is that the enum contains keys for everything, queries, request bodies, responses, etc. I'll leave it as is for now, perhaps this class won't be imported anywhere except the helpers that we will develop (if necessary, we can modify it there, or create subclassses `QueryKey`, `ResponseKey`, `SearchKey`, etc.).

Reply:
I agree with Nico regarding a more expressive name like `IndexerKey` or similar so it gives a hint of what the class is referring to.